### ISSUE TITLE
Service schema extension to restrict by network stack

### ISSUE THREAD

**Issue by jcpunk** [firewalld/firewalld#402]
This is mostly a sanity feature.  But it would be nice to be able to further restrict `dhcp*.xml` to only the relevant network stack (IPv4 or IPv6).  In practice I'm not sure there is much harm in listening on the wrong stack, but it does also expose the service over there which isn't ideal.

**Comment by erig0** [comment]
I agree it would be nice to have a `family` element that restricted to ipv4/ipv6. I wonder if the following works:
```
<destination ipv4="0.0.0.0/0" />
```
